Shapes.java

```diff
@@ -30,15 +33,54 @@ public static void rectangle(int length2,int width2, int areaRectangle){
  System.out.println("The area of the rectangle is " + areaRectangle);
 
 } //close method for rectangle
-public static void circle(double radius, double area){
+public static void circle(int radius, double area){
  Scanner input = new Scanner(System.in);
  System.out.println("What is the radius of the circle?");
- radius = input.nextDouble();
+ radius = input.nextInt();
 
  area = Math.PI * radius * radius;
 
  System.out.println("The area of the circle " + area);
-} // close mthod for circle
- }//close class
+} // close method for circle
 
```

Comment:
For all these methods calculating the volume, can you think of one way to keep your square, circle, etc. methods from before and have one method that gets the volume for all of those?

Reply:
So I should create one method that have no calculation? just prints out the volume?

Reply:
No... Remember static methods can call _other_ static methods. Does that help?